packages/api/src/services/market-prices.ts

```diff
@@ -0,0 +1,55 @@
+import bluebird from "bluebird";
+import { AppState } from "..";
+import { parseUnits, nowS } from "../libs/utils";
+
+type Config = undefined;
+
+type Dependencies = Pick<AppState, "zrx" | "marketPrices" | "collateralAddresses" | "syntheticAddresses">;
+
+// market prices are pulled from the 0x matcha api
+export default function (config: Config, appState: Dependencies) {
+  // these prices will be quoted against usdc by default, but can be specified as address or symbol
+  const { zrx, marketPrices, collateralAddresses, syntheticAddresses } = appState;
+  // this is hardcoded for now since it differs from the standard currency symbol usd
+  const currency = "usdc";
+
+  // does not do any queries, just a helper to mutate the latest price table
```

Comment:
I might be missing something that came in a previous PR, but where does the fetching actually happen?

Reply:
yea, there was a previous pr to this https://github.com/UMAprotocol/protocol/pull/3203.  This uses the 0x api